src/UrlParams.ts

```diff
@@ -408,6 +405,22 @@ export const getUrlParams = (
       };
   }
 
+  const intentDerivedConfiguration: IntentDerivedConfiguration = {};
```

Comment:
If I was being pedantic, this is more like intentAndPlatformDerivedConfiguration. The intent derived configuration is what you are doing above :)

Reply:
I like this and i dont see any harm in having a really long variable name here.